### PR TITLE
Switch to dcos-commons base docker image.

### PR DESCRIPTION
**Pull request by porridge**
https://github.com/mesosphere/spark-build/pull/297
removed requirements.txt, breaking the instructions.
This change fixes them.

### CODE REVIEW

File: scale-tests/README.md [hunk 1]
```diff
@@ -51,10 +51,9 @@ Cluster requirements:
 ```bash
 your-machine $ git clone git@github.com:mesosphere/spark-build.git
 your-machine $ cd spark-build
-your-machine $ docker run -it --net=host -v $(pwd):/spark-build -v </path/to/soak/key>:/ssh/key:ro mesosphere/spark-build:latest
-docker-container # cd /spark-build
-docker-container # pip3 install -r tests/requirements.txt
-docker-container # export PYTHONPATH=$(pwd)/testing:$(pwd)/spark-testing
+your-machine $ docker build -t mesosphere/dcos-commons:${LOGNAME}-spark scale-tests/
```
Comment (mpereira) [inline scale-tests/README.md]: What is `LOGNAME`?

File: scale-tests/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# TODO(mowsiany): the following image should match the version of scripts under
+# /tests
+FROM mesosphere/dcos-commons:latest
+ADD requirements.txt /tmp/tests-requirements.txt
+RUN pip3 install -r /tmp/tests-requirements.txt
+ENV PYTHONPATH="/spark-build/testing:/spark-build/spark-testing"
```
Comment (elezar) [inline scale-tests/Dockerfile]: Are we overwriting `PYTHONPATH` here? It's probably Ok though.

File: scale-tests/README.md (hunk 1 above)
Comment (elezar) [reply scale-tests/README.md]: Logged in username I believe: `$(whoami)`?

Comment (elezar) [reply scale-tests/README.md]: (At least that's what `echo $LOGNAME` shows for me)

Comment (porridge) [reply scale-tests/README.md]: Yes, IIRC $LOGNAME is POSIX, while $USER is not.

File: scale-tests/Dockerfile (hunk 2 above)
Comment (porridge) [reply scale-tests/Dockerfile]: Yes, the original instructions did that too.

**Pull request merged**